## 实验记录
虽然似乎成功地传递了数据给到linux这边，但是linux自己触发了页面异常，而且似乎卡顿了，没有办法成功继续往下跑

linux允许在自己运行的时候，因为页表异常而出现exception，但是我们得想一个办法让他能够跳转回来

我们确认一下相关的细节，这边还是比较繁琐的

我们首先是通过__agent_exit进入到user app中，之后由于时钟中断使然，我们会进入第一波__trap_entry，这个时候也许CSR_SSCRATCH中和tp寄存器中会有一些微妙的变化，得特别注意一下tp寄存器的使用

发现自己有一些误操作导致画蛇添足了，应该更加谨慎地对中断异常处理做好研究
- 先前我们为了提供一个共享内存区域来给agent传递参数来用，这个区域也许不需要额外添加，而是直接尝试放到原本kernel处理异常时，这些参数压栈的位置就行了，这样会非常优雅！